.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```diff
@@ -181,10 +185,15 @@ jobs:
       - name: (Artifact re-build) Build Webpack
         if: steps.download_artifact_1.outcome == 'failure'
         run: python -m scripts.build --prod_env
-      - name: Run E2E Test ${{ matrix.suite }}
-        run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_e2e_tests --skip-install --skip-build --suite=${{ matrix.suite }} --prod_env
-        env:
-          VIDEO_RECORDING_IS_ENABLED: 0
```

Comment:
What happened to this line? Was removing it intentional?

Reply:
Oh, yeah, it’s equivalent from our code perspective to not assign the variable and to assign it to zero.

I can leave it there if you want